src/pages/patientView/PatientViewGenePanelModal/PatientViewGenePanelModal.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { GenePanel } from 'shared/api/generated/CBioPortalAPI';
+import { observer } from 'mobx-react';
+import GenePanelModal from 'shared/components/GenePanelModal/GenePanelModal';
+import GenesList from './GenesList';
+import styles from './styles.module.scss';
+
+interface IPatientViewGenePanelModalProps {
+    genePanel: GenePanel;
+    show: boolean;
+    onHide: () => void;
+    columns?: number;
+}
+
+@observer
+export default class PatientViewGenePanelModal extends React.Component<
+    IPatientViewGenePanelModalProps,
+    {}
+> {
+    render() {
+        return (
+            <GenePanelModal
+                panelName="Gene Panel"
```

Comment:
ah okay, now I see why you passed in the gene panel name in your previous version. because the `GenePanel` object doesn't have a `name` attribute. sorry about that. but just want to make sure it's okay to use this generic "Gene Panel" name instead of passing in the exact name

Reply:
You were right before, gene panel name could be retrieved from `GenePanel.genePanelId`. Based on the mockup, the generic "Gene Panel" is just the header of the modal, while the exact gene panel name is placed in the modal body.
This is how the gene panel modal looks:
![Screenshot from 2019-10-30 11-14-02](https://user-images.githubusercontent.com/31291004/67849301-a1890d80-fb06-11e9-9fb3-b17a0eb66f7b.png)

Reply:
ah okay, sounds good thanks for clarifying.